### PR TITLE
Add several bug fixes to branch

### PR DESCRIPTION
**Pull request by djames1-hub**

### CODE REVIEW

File: analysis.js [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
+	let args = process.argv.slice(2);
 
-	if( args.length == 0 )
+	if( args.length === 0 )
 	{
-		args = ["analysis.js"];
+		args = ['analysis.js'];
 	}
-	var filePath = args[0];
+	let filePath = args[0];
```
Comment (Jcorrido) [inline analysis.js:14]: lots of variables getting changed from var to let

File: analysis.js [hunk 2]
```diff
@@ -45,12 +46,12 @@ function FunctionBuilder()
...
 		console.log(
 		   (
-		   	"{0}(): {1}\n" +
-		   	"============\n" +
-			   "SimpleCyclomaticComplexity: {2}\t" +
-				"MaxNestingDepth: {3}\t" +
-				"MaxConditions: {4}\t" +
-				"Parameters: {5}\n\n"
+		   	'{0}(): {1}\n' +
+		   	'============\n' +
```
Comment (Jcorrido) [inline analysis.js:50]: cool quote change

**Review by Jcorrido — CHANGES_REQUESTED** [review]
Liked the changes, seemed to just secure the code by making more variables and functions constants

File: analysis.js (hunk 1 above)
Comment (djames1-hub) [reply analysis.js:14]: Necessary change since var variable should be at the top of function.

File: analysis.js (hunk 2 above)
Comment (djames1-hub) [reply analysis.js:50]: Uniformity for string notation

File: analysis.js [hunk 3]
```diff
@@ -71,26 +72,26 @@ function FileBuilder()
...
+			  '~~~~~~~~~~~~\n'+
+			  'ImportCount {1}\t' +
+			  'Strings {2}\n'
 			).format( this.FileName, this.ImportCount, this.Strings ));
 	}
 }
 
 // A function following the Visitor pattern.
 // Annotates nodes with parent objects.
-function traverseWithParents(object, visitor)
```
Comment (djames1-hub) [inline analysis.js:84]: Change function declaration notation from old notation to new arrow function notation

**Pull request merged**